docs/reference/semantic-segmentation.md

```diff
@@ -0,0 +1,157 @@
+  - **drop_last**: *bool, default=True*  
+    Drop last data point if a batch cannot be filled. 
+  - **pin_memory**: *bool, default=False*  
+    Pin memory in dataloader. 
+  - **num_workers**: *int, default=4*  
+    Number of workers in dataloader. 
+  - **num_classes**: *int, default=12*  
+    Number of classes to predict among. 
+  - **crop_height**: *int, default=720*  
+    Input image height.
+  - **crop_width**: *int, default=960*  
+    Input image width.
+  - **context_path**: *str, default='resnet18'*  
+    Context path for the bisenet model.
+
+
+#### `BisenetLearner.fit`
+```python
+BisenetLearner.fit(self, dataset, val_dataset, silent, verbose)
+```
+
+This method is used for training the algorithm on a train dataset and validating on a val dataset.
+
+Parameters:
+  - **dataset**: *Dataset*    
+    Training dataset.
+  - **val_dataset**: *Dataset, default=None*  
+    Validation dataset. If none is given, validation steps are skipped.
+  - **silent**: *bool, default=False*  
+    If set to True, disables all printing of training progress reports and other information to STDOUT.  
+  - **verbose**: *bool, default=True*  
+    If set to True, enables the maximum logging verbosity. 
+
+
+#### `BisenetLearner.eval`
+```python
+BisenetLearner.eval(self, dataset, silent, verbose)
+```
+This method is used to evaluate a trained model on an evaluation dataset.
+Returns a dictionary containing stats regarding evaluation.  
+Parameters:
+  - **dataset**: *Dataset*  
+    Dataset on which to evaluate model.
+  - **silent**: *bool, default=False*  
+    If set to True, disables all printing of training progress reports and other information to STDOUT.  
+  - **verbose**: *bool, default=True*  
+    If set to True, enables the maximum logging verbosity. 
+
+
+#### `BisenetLearner.infer`
+```python
+BisenetLearner.infer(self, img, spath)
+```
+
+This method is used to perform segmentation on an image
+Returns a `engine.target.Heatmap` object.
+
+Parameters:
+  - **img**: *Image*  
+    Image to predict a heatmap.
+  - **spath**: *str, default=None*     
+    Path to save an additional output as image.  
+
+
+#### `BisenetLearner.download`
+```python
+BisenetLearner.download(self, path, mode, verbose, url)
+```
+
+Download pretrained models and testing images to path.
+
+Parameters:
+- **path**: *str*  
+  Path to metadata file in json format or to weights path.
+- **mode**: *{'pretrained', 'testingImage'}, default='pretrained'*  
+  If *'pretrained'*, downloads a pretrained segmentation model. If *'testingImage'*, downloads an image to perform inference on. 
+- **verbose**: *bool default=True*  
+  If True, enables maximum verbosity.
+- **url**: *str, default=OpenDR FTP URL*   
+  URL of the FTP server.
+
+
+#### `BisenetLearner.save`
+```python
+BisenetLearner.save(self, path, verbose)
+```
+
+Save model weights and metadata to path.
+
+Parameters:
+- **path**: *str*  
+  Directory in which to save model weights and meta data.
```

Comment:
You should try to use the same spelling of words within the documentation.
At line 133 there is "metadata"
```suggestion
  Directory in which to save model weights and metadata.
```